src/components/_modal/pages/loanerform/LoanerForm.js

```diff
@@ -112,21 +115,21 @@ export function UserParamsForm({ branch, initial, onSubmit, mode, originUrl }) {
               </Text>
               {userParameterType === "userMail" ? (
                 <Email
+                  valid={validMail}
                   value={state.userMail || ""}
-                  onChange={(e, { message }) => {
+                  onChange={(e) =>
                     setState({
                       ...state,
                       [userParameterType]: e?.target?.value,
-                    });
-                    setEmailMessage(message);
-                  }}
+                    })
+                  }
                   dataCy={`input-${userParameterType}`}
                   placeholder={
                     hasTranslation(placeholderTranslation)
                       ? Translate(placeholderTranslation)
                       : ""
                   }
-                  required
+                  required={true}
```

Comment:
hvis den altid er true er  der vel ingen grund til at den er der

Reply:
den er false per default
 du har ret, de to steder, hvor vi bruger det hen, er den true, så jeg laver true som default værdi 👍